src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisClientLibConfiguration.java

```diff
@@ -1139,6 +1252,28 @@ public KinesisClientLibConfiguration withMaxGetRecordsThreadPool(final int maxGe
     }
 
     /**
+     *
```

Comment:
You are missing one more config, i.e. max number of records in the cache.

Reply:
I think this is the maxRecords config, which is been configured at line 922.

Reply:
That is different than max number of records in the cache. This is the number of records the cache can hold versus the maxRecords is the max records each get call can return. This is the maxRecordsCount variable in the PrefetchGetRecordsCache.